pkg/apis/k3k.io/v1alpha1/types.go

```diff
@@ -338,10 +334,16 @@ type ClusterSet struct {
 // ClusterSetSpec defines the desired state of a ClusterSet.
 type ClusterSetSpec struct {
 
-	// DefaultLimits specifies the default resource limits for servers/agents when a cluster in the set doesn't provide any.
+	// Quota specifies the resource limits for clusters within a clusterset.
 	//
 	// +optional
-	DefaultLimits *ClusterLimit `json:"defaultLimits,omitempty"`
+	Quota *v1.ResourceQuotaSpec `json:"quota,omitempty"`
+
+	// Limit specifies the default value for a limitRange that will be applied to all pods within
+	// the clusterSet, if not specified a default value will be applied.
```

Comment:
Maybe we need to update the doc, because the LimitRange can be used also to set the min/max resource requests.

Wdyt about something like this?

```suggestion
	// Limit specifies the LimitRange that will be applied to all pods within the ClusterSet
	// to set defaults and constraints (min/max)
```

Reply:
Yes you are correct, I will update the docs as well